packages/router/package.json

```
@@ -21,7 +21,8 @@
   },
   "homepage": "https://github.com/angular/angular/tree/main/packages/router",
   "dependencies": {
-    "tslib": "^2.3.0"
+    "tslib": "^2.3.0",
+    "@types/dom-navigation": "^1.0.5"
```

Since these types are not exported in the public API, and don't show up in the package anywhere as they are built-ins in the browser, you can depend on it under `devDependencies`

```suggestion
    "tslib": "^2.3.0"
  },
  "devDependencies": {
    "@types/dom-navigation": "^1.0.5"
```